styles/about.css

```diff
@@ -0,0 +1,70 @@
+main {
+  position: relative;
+  top: 10vw;
+}
+
+body {
+  border: none;
+}
+
+.background {
+  position: fixed;
+  left: 0;
+  top: 7.2vw;
+  height: 100%;
+  z-index: 2;
+  opacity: 1;
+}
+
+.say_this h1 {
+  text-align: left;
+  padding-left: 5vw;
+  margin: 5vw;
+  margin-top: 0;
+  border-bottom: solid;
+  border-width: 60%;
+}
+
+
+.articles h1 {
+  z-index: 1;
+}
+
+.say_this {
+  z-index: 2;
+  display: block;
+  position: relative;
+  width: 80%;
+  text-align: center;
+  background: #EBF7F7;
+  opacity: .8;
+  padding-top: 5vw;
+  padding-bottom: 5vw;
+  margin: 5vw 0;
+  margin: 5vw;
+}
+
+
+
+.say_this article:nth-child(n) {
```

Comment:
you actually can leave off the `:nth-child(n)` bit! when you just have `n` as the parameter here, it just targets every article